### ISSUE TITLE
swap error - ARTWORK NOT FOR SALE - i paid but did not get swapped - please refund or fix

### ISSUE THREAD

**Issue by joshuabarrigada** [hicetnunc2000/hicetnunc#559]
i am wasting money, i spent on swapping twice, and it charged me but resulted in an error,

i tried again, and minted again, and swapped again,

same exact error

<img width="341" alt="Screen Shot 2021-04-22 at 11 37 56 AM" src="https://user-images.githubusercontent.com/82986721/115641053-ade8d200-a35b-11eb-8c76-ed5cc09cba92.png">
<img width="819" alt="Screen Shot 2021-04-22 at 11 37 25 AM" src="https://user-images.githubusercontent.com/82986721/115641061-b0e3c280-a35b-11eb-831b-e6323c066b37.png">
<img width="815" alt="Screen Shot 2021-04-22 at 11 37 17 AM" src="https://user-images.githubusercontent.com/82986721/115641068-b3461c80-a35b-11eb-90ea-657623a4847e.png">

**Comment by mescalina** [comment]
Refunds are not a concept well suited with decentralization. It's an experimental open-source platform, you use it at your own risk..

**Comment by claudiabraileanu** [comment]
I started today on HEN but I think I have the same issue... I was able to mint but it failed on swap with these error messages (from tzkt.io). What would be the solution for now? To burn and mint again? Could this have something to do with the uploaded file? I uploaded an mp4 file / 30 Mb. Maybe this is not the place to ask but is there a max file size for different file formats? Thank you

![error_call_swap](https://user-images.githubusercontent.com/48982503/115762625-5888e100-a3ac-11eb-8e14-921298ba95e4.jpg)
![error_call_transfer](https://user-images.githubusercontent.com/48982503/115762629-59217780-a3ac-11eb-9dce-d6b85c8b3bd4.jpg)
